src/policy/fees.h

```diff
@@ -174,8 +174,8 @@ class CBlockPolicyEstimator
      * invalidates old estimates files. So leave it at 1000 unless it becomes
      * necessary to lower it, and then lower it substantially.
      */
-    static constexpr double MIN_BUCKET_FEERATE = 1000;
-    static constexpr double MAX_BUCKET_FEERATE = 1e7;
+    static constexpr double MIN_BUCKET_FEERATE = 100;
```

Comment:
Per comment above, does it make sense to go lower, or is this as low as we'd likely ever want it?

Reply:
Lowering the fee 10x at the same time makes spamming UTXO set 10x cheaper. And due to blinding, filtering the spam-UTXO out will be difficult. My opinion is that lowering the fee should have come with excise on outputs, so that creating txs with a lot of outputs to spam the UTXO set would still be costly.

Reply:
@dgpv so this is just the bucket size of course. AJ has an (abandoned) PR in Core where he lowers them to 8. I talked with Lawrence about also doing that, but it seems like we can just as well lower this the next time we lower the minrelay fee. We need a new release to lower the relay fee anyhow, so.

Reply:
Sorry, my comment is probably misplaced -- it is about lowering fees in general, not about the fee estimator